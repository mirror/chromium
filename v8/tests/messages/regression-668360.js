// Does not print the right error message. When
// printing the error message the string is terminated
// when the 0-character is reached. 

var string = 'a' + String.fromCharCode(0) + 'xx';
string.doesNotExist()
