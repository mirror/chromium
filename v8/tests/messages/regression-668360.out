regression-668360.js:6: Uncaught TypeError: Object a xx has no method 'doesNotExist'
string.doesNotExist()
       ^
